### PR TITLE
Added New Compiler Resource to C++

### PR DESCRIPTION
**Pull request by Prasanthpadp**
Updated a C++ online compiler list with a free and easy to to use online resource.

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: more/free-programming-playgrounds.md [hunk 1]
```diff
@@ -72,6 +72,7 @@
 * [LabStack](https://code.labstack.com/cpp)
 * [OnlineGDB](https://www.onlinegdb.com/online_c++_compiler)
 * [SoloLearn](https://code.sololearn.com/cpp)
+* [InterviewBit](https://www.interviewbit.com/online-cpp-compiler/)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:75]: Move it between line 71-72 😉

**Review by Prasanthpadp — COMMENTED** [review]
Changes made. Please check it.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**